packages/smooth_app/lib/l10n/app_en.arb

```diff
@@ -1007,7 +1007,7 @@
         }
     },
     "camera_mode_file_based": "Safe mode (by default)",
-    "camera_mode_bytes_array_based": "Quick mode",
+    "camera_mode_bytes_array_based": "糖",
```

Comment:
non

Reply:
For the record I don't think we use that label anymore.